extensions/healthie/validation/getPatient.zod.ts

```diff
@@ -0,0 +1,64 @@
+import { type E164Number } from 'libphonenumber-js'
+import {
+  DateTimeSchema,
+  E164PhoneValidationSchema,
+} from '../../../lib/shared/validation'
+import { type ActivityEvent } from '../../../lib/types/ActivityEvent'
+import { type GetUserQuery } from '../gql/sdk'
+
+interface ValidateResult {
+  data: { phoneNumber?: E164Number; dob?: string }
+  events?: ActivityEvent[]
+}
+export const validateGetPatient = (
+  user: GetUserQuery['user']
+): ValidateResult => {
+  const phoneValidationResult = E164PhoneValidationSchema.safeParse(
+    user?.phone_number
+  )
+  const dobValidationResult = DateTimeSchema.safeParse(user?.dob)
+
+  return {
+    data: {
+      dob: dobValidationResult.success ? dobValidationResult.data : undefined,
+      phoneNumber: phoneValidationResult.success
+        ? phoneValidationResult.data
+        : undefined,
+    },
+    events:
+      phoneValidationResult.success && dobValidationResult.success
+        ? undefined
+        : ([
+            ...(phoneValidationResult.success
+              ? []
+              : [
+                  {
+                    date: new Date().toISOString(),
+                    text: {
+                      en: "Phone number from Healthie not stored because it isn't a valid E.164 phone number",
+                    },
+                    error: {
+                      category: 'WRONG_DATA',
+                      message:
+                        "Phone number from Healthie not stored because it isn't a valid E.164 phone number",
```

Comment:
```suggestion
                        `Phone number from Healthie (${String(user?.phone_number)}) not stored because it isn't a valid E.164 phone number`,
```